src/voided-disputes.js

```diff
@@ -0,0 +1,37 @@
+import { networks, RINKEBY_COURT, RINKEBY_USABILITY_COURT } from './networks'
+import { getNetworkName } from './lib/web3-utils'
+import env from './environment'
+
+const DEFAULT_VOID_TEXT = 'This is a text'
+const DEFAULT_VOID_LINK = ''
+
+export const VOIDED_DISPUTES = {
```

Comment:
Should we avoid exporting this for now, since all internal code should be using `getVoidedDisputesByCourt()`?

Reply:
Ah yes, initially i was importing it from another file and forgot to remove the export after implementing `getVoidedDisputesByCourt `